src/index.html

```diff
@@ -3,11 +3,398 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="./images/favicon-32x32.png"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    <header class="page__header header">
+      <div class="header__wrapper">
+        <section class="header__control-area">
+          <a href="#" class="header__logo">
+            <img
+              src="./images/bose.svg"
+              alt="bose-icon"
+              class="header__logo-image"
+            >
+          </a>
+          <div class="header__icons">
+            <a
+              href="tel:+1 234 555-55-55"
+              class="header__icons-link"
+            >
+              <img
+                src="./images/phone.svg"
+                alt="phone"
+                class="header__icons-phone"
+              >
+            </a>
+
+            <a href="#menu" class="header__icons-toggler"></a>
+          </div>
+        </section>
+
+        <div class="header__title-wrapper">
+          <div class="header__sound-waves"></div>
+
+          <h1 class="header__title">
+            The world shades.
+            <br>
+            Your music shines.
+          </h1>
+        </div>
+      </div>
+
+      <nav
+        class="header__menu menu"
```

Comment:
try to prevent page to scroll when menu is displayed
![scroll under the menu](https://user-images.githubusercontent.com/39308717/162919010-e95d5bc5-bbb9-46e1-8fea-87e5fe260619.png)
> 18. To disable page scrolling under the menu ...